packages/core/addon/components/navi-visualizations/goal-gauge.ts

```diff
@@ -146,14 +150,14 @@ export default class NaviVisualizationsGoalGaugeComponent extends Component {
   /**
    * @property {Object} - general gauge configuration
    */
-  @computed('options.{baselineValue,goalValue}', 'prefix', 'unit')
+  @computed('args.options.{baselineValue,goalValue}')
```

Comment:
Maybe we can just remove most of the aliases to `config.*` items, it seems odd that we reference them in so many ways, then all the computed keys/references can use `this.config.*`